src/coreclr/jit/codegenlinear.cpp

```diff
@@ -2695,24 +2695,24 @@ void CodeGen::genEmitterUnitTests()
     // Add NOPs at the start and end for easier script parsing.
     instGen(INS_nop);
 
-    bool unitTestSectionAll = (u16_strstr(unitTestSection, W("all")) != nullptr);
```

Comment:
Can you please check if this is the last usage of the `u16_strstr` function? In fact for any of the `u16_` prefixed functions please remove them if this is the last use.

Reply:
Doesn't look like any of the removed `u16_` functions were the last uses of those functions.